dashboards-observability/public/components/explorer/hooks/use_fetch_events.ts

```diff
@@ -126,6 +132,29 @@ export const useFetchEvents = ({
     });
   };
 
+  const getLiveTail = (query: string = '', errorHandler?: (error: any) => void) => {
+    const cur = queriesRef.current;
+    const searchQuery = isEmpty(query) ? cur![requestParams.tabId][FINAL_QUERY] : query;
+    fetchEvents({ query: searchQuery }, 'jdbc', (res: any) => {
+      if (!isEmpty(res.jsonData)) {
+        if (!isEmpty(responseRef.current)){
+          res.jsonData = res.jsonData.concat(responseRef.current.jsonData);
+          res.datarows = res.datarows.concat(responseRef.current.datarows);
+          res.total = res.total + responseRef.current.total;
+          res.size = res.size + responseRef.current.size;
+        }
+        // send only first 200 data log arrays
```

Comment:
limit this in the backend request instead of here?

Reply:
it only fetches the new live logs and prepends it to the old data so i'm just slicing off the old data( already fetched and stored) to make space new data logs to be displayed.